docs/operators/operator-node/node-setup/best-practices.md

```diff
@@ -0,0 +1,224 @@
+---
+sidebar_label: 'Best Practices'
+sidebar_position: 3
+---
+
+# Best Practices for Running SSV Node
+
+This guide outlines key recommendations to ensure the best Performance and Correctness of your node. The guide will be updated as we get new information.
+
+The page is divided into three parts:
+- [**Introduction**](#introduction) - understanding the Performance, Correctness, and their key factors.
+- [**Major impact**](#major-impact) - vital tips, more obvious.
+- [**Minor impact**](#minor-impact) - fine-tuning tips, nice-to-have, less obvious.
+
+
+## **Introduction**
+
+Performance you see on [SSV Explorer](https://explorer.ssv.network/) is essentially participation rate - how many duties your node participated in. Correctness refers to how timely and accurate duties are, and it affects validators' rewards.
+
+Several key factors influence this:
+- **Setup Configuration:** Software used to manage nodes, configured ports and the firewall help eliminate delays.
+- **Network Throughput & Latency:** Minimal network delay is critical, especially for production setups.
+- **Hardware Resources:** Adequate CPU, sufficient RAM, and fast, reliable storage are necessary.
+
+## **Major impact**
+More obvious, more impactful.
+
+### **For Home Setups**
+- **Hardware:** Usual setup has 4- to 8-core CPU (focus on single-thread performance) and 32GB RAM.
+- **Disk:** NVMe SSDs are strongly recommended. 2TB size should serve you until ~2026. You can have 4TB to be on the safe side. Stick to `ext4` filesystem, as it has proven more performant. 
+- **Reliability:** Use UPS and keep machine well-ventilated. Sudden power loss or thermal throttling can both cause downtime or missed duties.
+- **Internet Connectivity:** Ensure your ISP doesn't impose strict data caps. Choose a plan with at least 10 Mbps upload speed, but latency and reliability make the difference.
+
+### **For Enterprise Grade**
+- **CPU frequency is a major factor**. Fast cores with moderate-to-high core count are the recipe. Ensure each server can turbo boost to high frequency, yet avoid agressive overclocking.
+- **Disk and Redundancy:** NVMe SSDs are mandatory. Many use RAID1 mirroring or have failover nodes ready in case of disk failure. Recommended to use ext4 if unsure about ZFS's complexities. Improving read/write speed has one of the greatest impacts.
+- **RAM Considerations:** Usually RAM is not the limiting factor. Abundant RAM enables disk caching for the execution client database – reducing disk reads and writes. ECC memory is advisable to protect against bit flips in long-running servers. 
+- **Form Factor and Data Center:** Run on a mix of bare metal servers in colocation data centers and cloud VM instances. In either case, redundancy across hardware and geographic distribution is a best practice. 
```

Comment:
"a mix of" sounds a bit misleading. It almost sounds like they should have SSV node on bare metal, CL on cloud, and EL somewhere else. Can you be more specific?